rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -21,9 +21,25 @@
     cache_valid_time: 600
   with_items: maas_apt_packages
 
-- name: Install pip packages
+- name: Create venv dir for maas
```

Comment:
Heh, so you don't actually need this task since the first use of the pip module with `virtualenv:` will do that for you. Plus I think this will *always* create the virtualenv even when `maas_venv_enabled|bool` evaluated to false (because there's no condition here.